doc/lua_api.txt

```diff
@@ -5288,6 +5288,16 @@ Call these functions only at load time!
 * `minetest.register_on_item_eat(function(hp_change, replace_with_item, itemstack, user, pointed_thing))`
     * Called when an item is eaten, by `minetest.item_eat`
     * Return `itemstack` to cancel the default item eat response (i.e.: hp increase).
+* `minetest.register_on_item_pickup(function(itemstack, picker, obj, time_from_last_punch,  ...))`
+    * Called when an item is picked up, after `itemdef.on_pickup`.
+    * Function is added to `minetest.registered_on_item_pickups`.
+    * Oldest functions are called first.
+    * Parameters:
+      * `itemstack`: The item to be picked up. Do not modify!
+      * `picker`: ObjectRef or `nil`
+      * `obj`: The dropped item (a `"__builtin:item"` luaentity ObjectRef).
+      * `time_from_last_punch, ...`: Other parameters from `luaentity:on_punch`.
+    * Return `true` to cancle pick-up. Remaining functions are then not called.
```

Comment:
```suggestion
    * Return `true` to cancel the pick-up . Remaining functions are then not called.
```
Also an inconsistency: Why does `true` to cancel it? `on_pickup` lets it continue on `true`.

Reply:
Done. But this needs fixes in `core.run_callbacks`. And returning nothing results in cancellation.